Project.toml

```
@@ -3,13 +3,10 @@ uuid = "276daf66-3868-5448-9aa4-cd146d93841b"
 version = "0.8.0"
 
 [deps]
-BinDeps = "9e28174c-4ba2-5203-b857-d8d62c4213ee"
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+OpenSpecFun_jll = "efe28fd5-8261-553b-a9e1-b2916fc3738e"
 
 [compat]
-BinaryProvider = "≥ 0.5.0"
-julia = "≥ 0.7.0"
+julia = "≥ 1.3"
```

```suggestion
julia = "1.3"
```
Also need compatibility for the openspecfun version

---

Is the `≥` unnecessary?

---

https://github.com/JuliaRegistries/General/blob/master/O/OpenSpecFun_jll/Versions.toml 

OpenSpecFun_jll  = "0.5.3" for the minimum Ok ?

---

`≥` means literally any Julia version at or after 1.3.0. Omitting it uses the implicit caret specifier, which gives you only semver-compatible versions. That's important because we don't want this version to be installable on a theoretical future Julia 2.0, for example.

---

> OpenSpecFun_jll = "0.5.3" for the minimum Ok ?

Yep that works

---

One more question, does setting `OpenSpecFun_jll = "0.5.3"` allow any version of up to and including `1.x` of the package?

---

See https://julialang.github.io/Pkg.jl/v1/compatibility/. In this case, it fixes OpenSpecFun_jll to 0.5.3, which is good, since we want to ensure we're using a specific version of the openspecfun library.